README.md

```diff
@@ -134,6 +135,90 @@ import "@asyncapi/react-component/styles/default.css";
 import "@asyncapi/react-component/styles/default.min.css";
 ```
 
+### Custom styles
+The asyncapi react-component does not set any global fonts. This provides the option to define your custom font-family and related font styling.
+This can be done by defining the styles in a file or inline using by adding a `<style>` tag in the `<head>` section of the page where you are using asyncapi react-component.
```

Comment:
```suggestion
The AsyncAPI React component does not set any global fonts and any global reset styles. This provides the option to define your custom `font-family` and other styling.

This can be done by defining the styles in a file or inline using by adding a `<style>` tag in the `<head>` section of the page where you are using AsyncAPI React component.
```